server/sheet.go

```diff
@@ -0,0 +1,199 @@
+
+		sheet, err := s.SheetService.CreateSheet(ctx, sheetCreate)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to create sheet").SetInternal(err)
+		}
+
+		if err := s.composeSheetRelationship(ctx, sheet); err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch created sheet relationship").SetInternal(err)
+		}
+
+		c.Response().Header().Set(echo.HeaderContentType, echo.MIMEApplicationJSONCharsetUTF8)
+		if err := jsonapi.MarshalPayload(c.Response().Writer, sheet); err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to marshal create sheet response").SetInternal(err)
+		}
+		return nil
+	})
+
+	g.GET("/sheet", func(c echo.Context) error {
+		ctx := context.Background()
+		sheetFind := &api.SheetFind{}
+		creatorID := c.Get(getPrincipalIDContextKey()).(int)
+		sheetFind.CreatorID = &creatorID
+
+		if rowStatusStr := c.QueryParam("rowstatus"); rowStatusStr != "" {
+			rowStatus := api.RowStatus(rowStatusStr)
+			sheetFind.RowStatus = &rowStatus
+		}
+
+		if instanceIDStr := c.QueryParams().Get("instanceId"); instanceIDStr != "" {
+			instanceID, err := strconv.Atoi(instanceIDStr)
+			if err != nil {
+				return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("ID is not a number: %s", c.QueryParam("instanceId"))).SetInternal(err)
+			}
+			sheetFind.InstanceID = &instanceID
+		}
+
+		if databaseIDStr := c.QueryParams().Get("databaseId"); databaseIDStr != "" {
+			databaseID, err := strconv.Atoi(databaseIDStr)
+			if err != nil {
+				return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("ID is not a number: %s", c.QueryParam("databaseId"))).SetInternal(err)
+			}
+			sheetFind.DatabaseID = &databaseID
+		}
+
+		if visibility := c.QueryParam("visibility"); visibility != "" {
+			sheetFind.Visibility = &visibility
+		}
+
+		list, err := s.SheetService.FindSheetList(ctx, sheetFind)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch sheet list").SetInternal(err)
+		}
+
+		for _, sheet := range list {
+			if err := s.composeSheetRelationship(ctx, sheet); err != nil {
+				return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch sheet relationship: %v", sheet.Name)).SetInternal(err)
+			}
+		}
+
+		c.Response().Header().Set(echo.HeaderContentType, echo.MIMEApplicationJSONCharsetUTF8)
+		if err := jsonapi.MarshalPayload(c.Response().Writer, list); err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to marshal sheet list response").SetInternal(err)
+		}
+		return nil
+	})
+
+	g.GET("/sheet/:id", func(c echo.Context) error {
+		ctx := context.Background()
+		id, err := strconv.Atoi(c.Param("id"))
+		if err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("ID is not a number: %s", c.Param("id"))).SetInternal(err)
+		}
+
+		creatorID := c.Get(getPrincipalIDContextKey()).(int)
+		sheetFind := &api.SheetFind{
```

Comment:
Are we querying sheets where ID=id AND CreatorID=creatorID?

Reply:
no need add createID